### PR TITLE
array: slice

### PR DESCRIPTION
**Pull request by dhonx**

### CODE REVIEW

File: examples/arrays.md [hunk 1]
```diff
@@ -137,6 +137,28 @@ false
...
 
+## Slicing an array
+
+It's easy to slice an array in V. You can slice an array with the default 
+V slicing feature without having to call the `slice ()` method.
+The syntax is like this `my_array[start..end]`
+
+```go
+animals := ['lion', 'goose', 'chicken', 'turkey', 'tiger']
+poultry := [1..4]
```
Comment (vbrazo) [inline examples/arrays.md]:
```suggestion
poultry := animals[1..4]
```

**Comment by vbrazo** [comment]
I think we should merge the files `Array` and `Arrays Functions`.

**Comment by dhonx** [comment]
I think is not related to `function` :thinking:

**Review by vbrazo — APPROVED** [review]
Looks good 👍

File: examples/arrays.md (hunk 1 above)
Comment (dhonx) [reply examples/arrays.md]: hahaha... i forgot

**Pull request merged**